bitcoin/src/pow.rs

```diff
@@ -1652,27 +1595,11 @@ mod tests {
     #[test]
     #[should_panic]
     fn u256_overflowing_subtraction_panics() { let _ = U256::ZERO - U256::ONE; }
-
```

Comment:
trailing whitespace

Reply:
oooh, I must have skipper running the githook - my bad. Thanks

Reply:
The githook misses that whitespace - nooooooo